src/main/java/appeng/parts/CableBusContainer.java

```diff
@@ -848,6 +862,14 @@ private NBTTagCompound[] covertEC2Part(NBTTagCompound part, NBTTagCompound data)
                 data_new.setInteger("priority", data.getInteger("priority"));
                 data_new.setString("ACCESS", data.getString("access"));
                 NBTTagCompound fluid_filter = new NBTTagCompound();
+                NBTTagList upgrades = data.getTagList("upgradeInventory", TAG_COMPOUND);
+                NBTTagCompound upgradeNew = new NBTTagCompound();
+                upgradeNew.setTag("#0", upgrades.getCompoundTagAt(0));
```

Comment:
There is just one possible upgrade?

Reply:
Yes, see the screenshot in the linked issue